modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -3267,6 +3267,10 @@ CV_EXPORTS float EMD( InputArray signature1, InputArray signature2,
                       int distType, InputArray cost=noArray(),
                       float* lowerBound = 0, OutputArray flow = noArray() );
 
+CV_EXPORTS_AS(EMD) float wrapperEMD( InputArray signature1, InputArray signature2,
```

Comment:
@alalek Since ABI will not allow us to change the current signature (although it remains consistent), we are only left with either to overload or to create a new function.
The problem with overloading is that `cost` has a default argument (and we cannot leave Mat without initialization) and before it `lowerBound` exists in the list, therefore `lowerBound` needs a default argument as well. This causes ambiguity, and hence it is impossible to fix the issue without a rename!